libmobilecoin/src/chacha20_rng.rs

```diff
@@ -0,0 +1,144 @@
+use crate::common::*;
+use mc_util_ffi::*;
+use rand_chacha::ChaCha20Rng;
+use rand_core::{RngCore, SeedableRng};
+use std::{convert::TryInto, sync::Mutex};
```

Comment:
Nit: `TryInto` is now part of the standard preamble for Rust edition 2021.
```suggestion
use std::sync::Mutex;
```